### DISCUSSION TITLE
Persistent Timer

### DISCUSSION THREAD

**Discussion by harmonz** [PaulWoitaschek/Voice#1286] [Ideas]
**Add function of "_Persistent timer_" as a toggle along with "_Shake to reset_" in the sleep timer window.**
**_Purpose_:** The play button will always activate or maintain the sleep timer whether by the interface, notification, widget, or headphones.
_**Why this might matter:**_ Listening at night failing to reset sleep timer makes the audiobook a part of the dream state.  If the phone is out of reach to shake, the headphone play button will continue playback with sleep timer again. I may be the only person in the world who falls asleep listening to audiobooks (doubtful), but will always want a timed playback even while I'm hiking or shopping.  Restarting playback is easy while awake. Stopping playback is difficult while asleep.  I've never needed playback to go to the end of a 10 hour book nonstop.

**Comment by nschulzke** [comment]
I've just started using this app, and I would also like to see this feature. Has this been converted to a discussion because it's up for debate whether it should be implemented, or would you accept a PR for it?
